### PR TITLE
style: optimize cascader clear animation

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [x] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
- https://ant.design/components/cascader-cn/#components-cascader-demo-default-value

点开下拉框，点击 clear 按钮，可以看到，面板会先进行清除动画，再进行关闭动画。动画效果不连贯。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Optimize Cascader clear animation.|
| 🇨🇳 Chinese | 优化 Cascader 清除动画效果。          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/cascader/index.tsx [hunk 1]
```diff
@@ -354,8 +354,10 @@ class Cascader extends React.Component<CascaderProps, CascaderState> {
     e.preventDefault();
     e.stopPropagation();
     if (!inputValue) {
-      this.setValue([]);
       this.handlePopupVisibleChange(false);
+      setTimeout(() => {
```
Comment (afc163) [inline components/cascader/index.tsx:358]: 这个 setTimeout 要 clear 掉。

**Comment by afc163** [comment]
test cov

**Comment by xrkffgg** [comment]
Emmm，这次应该可以了~

**Pull request merged**